.build/CBT/build.props

```diff
@@ -66,9 +66,18 @@
   <UsingTask TaskName="RestoreModules" AssemblyFile="$(CBTCoreAssemblyPath)" />
 
   <Target Name="RestoreCBTModules"
-      Condition=" '$(CBTModulesRestored)' == '' "
+      Condition=" '$(CBTModulesRestored)' != 'true' "
       Inputs="$(CBTModuleRestoreInputs)"
       Outputs="$(CBTModulePropertiesFile)">
+    
+    <!-- TODO: What should this error message say?  The user would have already received this error (but the build continues and then this stops the build):
+
+    MSBUILD : error : Restoring CBT modules failed with an exit code of '1'.  More information about the problem including the output of the restoration command is
+    available when the log verbosity is set to Detailed.
+    
+    Is there a task that just stops the build?
```

Comment:
You can only fail during parse time by making evaluation fail such as an exception.  Example defining a property like this.

<EnableNuGetInstallAfterPack>$([System.Convert]::ToBoolean('Fail the Build here'))</EnableNuGetInstallAfterPack>

gives:
"d:\tfs\IS\ES\M\Product\Source\ProjectCop\Task\ProjectCop.csproj" (default target) (1) ->
  d:\tfs\IS\ES\M\Product\Source\ProjectCop\Task\ProjectCop.csproj(14,1): error MSB4184: The expression "[System.Convert]::ToBoolean(Fail the Build here)" cannot be evaluated. String was not recognized as a valid Boolean.